docs/reference/cli/filters.md

```diff
@@ -0,0 +1,73 @@
+# Using Integration and Filters in K8sGPT
```

Comment:
I would explain both filters and integrations separately first

* Filters are a way of selecting which resources you wish to be part of your default analysis
* Integrations are a way to add in additional resources to the filter list.